src/main/java/Calculator.java

```diff
@@ -0,0 +1,54 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.*;
+
+public class Calculator {
+
+    static int splitAndSum(String inputData) {
+        if (checkIfEmpty(inputData)) {
+            return 0;
+        }
+
+        List<String> elements = splitText(inputData);
+
+        return sumAll(elements);
+    }
+
+    private static boolean checkIfEmpty(String text) {
+        return  text == null || text.isEmpty();
+    }
+
+    private static List<String> splitText (String inputData){
+        String delimiter, realText;
+
+        final Matcher m = Pattern.compile("//(.)\n(.*)").matcher(inputData);
+        if (m.find()) {
+            delimiter = m.group(1);
+            realText = m.group(2);
+        } else {
+            delimiter = ",|:";
+            realText = inputData;
+        }
+
+        return Arrays.asList(realText.split(delimiter));
+    }
+
+    private static int sumAll (List<String> strings) {
+        return convertStringArrayToIntArray(strings).stream().mapToInt(Integer::intValue).sum();
+    }
+
+    private static List<Integer> convertStringArrayToIntArray(List<String> inputString) {
+        List<Integer> output = new ArrayList<>();
+        for (String el : inputString) {
```

Comment:
```suggestion
        for (String token : inputString) {
```
변수명를 줄이다 보면 의미 전달이 제대로 되지 않을 수 있어요!

Reply:
오 el은 사내에서 element의 약자로 많이 사용해왔었는데 너무 줄였나요ㅎㅎㅎ

Reply:
이번 과정에서는 변수명 하나하나 의미를 다 부여하면서 해보시죠 ㅎㅎ
element도 너무 추상화되어있는 단어입니다.